NEWS

```diff
@@ -118,13 +122,16 @@ Bug fixes/minor improvements:
   orphan children nodes in the launch tree.
 - Fix the help message when showing deprecated MCA param names to show
   the correct (i.e., deprecated) name.
+- Enable support for OpenIB BTL use across different IB subnets.
 - Fix a minor error in MPI_AINT_DIFF.
 - Fix bugs with MPI_IN_PLACE handling in:
   - MPI_ALLGATHER[V]
   - MPI_IALLTOALL*
   - MPI_[I][GATHER|SCATTER][V]
   - MPI_IREDUCE[_SCATTER]
   - Thanks to all the users who helped diagnose these issues.
+  JMS - WERE ANY OF THESE NOT FIXED in 2.0.X? See below for numerous
+  citations.
```

Comment:
Good catch; I missed the de-duplication.  I think ALLGATHER[V] and IALLTOALL* were cited below, but I didn't see [I][GATHER|SCATTER][V] or IREDUCE[_SCATTER].